Weather/Weather/Scenes/ListCityWeather/ListCityWeatherViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  CityManagermentScreenViewController.swift
+//  Weather
+//
+//  Created by minh duc on 2/25/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class ListCityWeatherViewController: UIViewController, StoryboardSceneBased {
+    @IBOutlet private weak var listCityCollectionView: UICollectionView!
+    
+    private struct Constant {
+        static let itemSize = CGSize(width: Constants.widthScreen/2, height: Constants.widthScreen/2)
+    }
+    
+    static let sceneStoryboard = UIStoryboard(name: Constants.nameStoryboard, bundle: nil)
```

Comment:
ném vào extension riêng nằm ở cuối file
tạo 1 `struct StoryBoards` để khởi tạo các storyboard nữa
```suggestion
    // MARK: - StoryboardSceneBased
extension ListCityWeatherViewController: StoryboardSceneBased {
    static var sceneStoryboard = StoryBoards.storyboardName
}
```